monkey/infection_monkey/utils/environment.py

```diff
@@ -8,11 +8,15 @@ def get_os() -> OperatingSystem:
     """
     Get the OperatingSystem of the current execution environment.
     """
+    # platform.system can return 'Linux', 'Windows', 'Darwin', 'Java'
+    # or empty string if the value can't be determined
     system = platform.system()
     if system == "Windows":
         return OperatingSystem.WINDOWS
-    if system == "Linux":
+    elif system == "Linux":
         return OperatingSystem.LINUX
+    elif system != "":
+        return OperatingSystem.ANY
```

Comment:
It's not "any", it's "unknown"

Reply:
It is "unkown" in case if it is ""

Reply:
Ahhh, my bad, it's `!=`. Then it's unsupported and should raise an error as it did.

Reply:
+1, should raise exception.